### ISSUE TITLE
Show "instructions per clock" for E- and P-cores

### ISSUE THREAD

**Issue by samuelfredrickson** [tlkh/asitop#37]
`powermetrics` reports this metric for each CPU core cluster, as well as an average for the entire SoC. I've found it interesting to see how this value fluctuates under various workloads, and think it would be neat to see it incorporated into `asitop`.

**Comment by NightMachinery** [comment]
What does this value show exactly? Is it better higher or lower? I find it very difficult to interpret.

**Comment by BitesPotatoBacks** [comment]
@NightMachinery It is a metric of how many instructions we executed by the cpu cluster, per clock cycle, during the metric sample.

Higher is better assuming you desire higher performance (from faster execution of instructions).

**Comment by NightMachinery** [comment]
@BitesPotatoBacks Some instructions do more work than others though. Perhaps not as much of an issue in ARM CPUs.

**Comment by BitesPotatoBacks** [comment]
@NightMachinery Of course, certain instructions may take multiple cycles to complete (but these timings are often tied to the efficiency of the microarch). Though, I do not know the exact timings of the instructions supported by current Apple Silicon.

In `powermetrics`, Instructions per clock should be an average value for the cluster that gives the metric. Longer execution times for instructions would theoretically decrease the value.

**Comment by sowa705** [comment]
This is a very interesting metric, Im pretty sure you can also get this metric on ryzen cpu's so it would be very interesting for comparisons between different CPU's
